scripts/processors/cams_processor.py

```
@@ -0,0 +1,343 @@
+            output_path = output_dir / f"{self.country.lower()}_cams_{date_range_str}_{timestamp_str}.csv"
+            
+        logger.info(f"Saving to {output_path}")
+        result.to_csv(output_path, index=False)
+        logger.info(f"Saved {len(result)} records to {output_path}")
+        
+        # Print summary
+        self.print_summary(result)
+        
+        return result
+        
+    def print_summary(self, df: pd.DataFrame):
```

Ruff (0.12.2)

Missing return type annotation for public function `print_summary`

Add return type annotation: `None`

(ANN201)